Cargo.toml

```diff
@@ -312,6 +314,7 @@ num-bigint = "0.3"
 num_cpus = "1.11"
 num-rational = { version = "0.3.1", features = ["serde"] }
 num-traits = "0.2.15"
+okapi = { git = "https://github.com/near/near-okapi-fork.git", rev = "fd7de89e130ab99a546f04e3faefcd53044f98d0", features = ["schemars-alpha"] }
```

Comment:
Add a comment with the reference to the PR to the upstream okapi and mention that we can get back to using the upstream crate as soon as there is shemars 1.0 (alpha or release) is supported.